sketches/CAN_haz_bus/CAN_haz_bus.ino

```diff
@@ -72,7 +72,7 @@ void logHexStr(INT32U num, char* prefix, int len)
 
 void setup()
 {
-    Serial.begin(500000);
+    Serial.begin(4000000);
```

Comment:
Yes, actually. This is left over from my playing around with the Uno a few weeks ago, but I determined that I could go up to 4 mbps with no ill effect